website/client/components/static/home.vue

```diff
@@ -598,11 +598,17 @@
           return;
         }
 
+        let partyInvite = '';
+        if (this.$route.query && this.$route.query.p) {
```

Comment:
@Alys @TheHollidayInn I don't understand where this p parameter is coming from... In the code I only found a reference to ?groupInvite, are we sure it's correct?

Reply:
If you looks at issue 9450, the link posted has the p parameter. Where are you looking?

Reply:
Here https://github.com/HabitRPG/habitica/blob/develop/website/server/controllers/api-v3/groups.js#L1028 (we'll have to change the static/front part too). That link in the issue doesn't work so I'm not sure if it's just wrong

Reply:
Ah, nice. Easy enough to change. Also see here: https://github.com/HabitRPG/habitica/blob/develop/website/server/controllers/api-v3/auth.js#L163

Reply:
So initially we used partyInvite, then expanded the feature to all groups so I created ?groupInvite but kept partyInvite there in case someone used an old invite link but all the emails we're sending out right now should only use `?groupInvite`